docs/source/user_guide/pipelines.ipynb

```diff
@@ -389,8 +392,8 @@
     "\n",
     "    def fit(self, X, y=None):\n",
```

Comment:
Updating demo code to be a transformer that handles woodwork :') necessary for fitting of pipeline below!